### PR TITLE
Add support for multi-value SuppressWarnings

### PR DESCRIPTION
**Pull request by Shredder121**

### CODE REVIEW

**Comment by timowest** [comment]
looks good

**Comment by johnktims** [comment]
This looks good, but I think it would be a good idea to add a test. Maybe at the end of  `JavaWriterTest`?

File: src/test/java/com/mysema/codegen/JavaWriterTest.java [hunk 1]
```diff
@@ -335,4 +335,12 @@ public void SuppressWarnings() throws IOException {
 
         match("/testSuppressWarnings", w.toString());
     }
+
+    @Test
+    public void SuppressWarnings2() throws IOException {
+        writer.suppressWarnings("all", "unused");
```
Comment (Shredder121) [inline src/test/java/com/mysema/codegen/JavaWriterTest.java]: Test added.

**Comment by johnktims** [comment]
That looks better. Thanks.

File: src/main/java/com/mysema/codegen/JavaWriter.java [hunk 2]
```diff
@@ -481,6 +481,11 @@ public JavaWriter suppressWarnings(String type) throws IOException {
         return line("@SuppressWarnings(\"" + type + "\")");
```
Comment (Shredder121) [inline src/main/java/com/mysema/codegen/JavaWriter.java]: a `line(String)` call is much faster than processing the annotation, so I'd rather keep the original.

Comment (johnktims) [inline src/main/java/com/mysema/codegen/JavaWriter.java]: Okay. That's fine.

**Pull request merged**